app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command, file_name, path = command.split()
```

Comment:
DRY - do not use `command.split()` twice

Reply:
what is mean DRY??